SEMANTIC_CONVENTIONS.md

```diff
@@ -28,33 +28,106 @@ We use certain data types for the properties we expect.
+| sources              | A list of all sources, line by line. This should include all URLs or other sources used.             | list\<text>                          | RECOMMENDED |
+| source.event.page    | the url of the page where the event was sourced from                                                 | url                                  | RECOMMENDED |
+| source.event.list    | an url to a list of events on the page where the event was sourced from (if available)               | url                                  | OPTIONAL    |
+| source.event.details | an url that leads to the details of the specific event on the page where the event was sourced from  | url                                  | RECOMMENDED |
+
+#### Usage explanation
+
+`MANDATORY`: This field is required for proper operation of boudicca.
+`RECOMMENDED`: This field is not required, but it is one of the core features enabling event sorting and filtering for publishers.
+`OPTIONAL`: This field can be used if the information is available.
+`DEPRECATED`: This field should no longer be used.
 
 #### Registration enum values
 
 * `free`: a free event which neither requires registration nor a ticket
 * `registration`: an event which requires a free registration
 * `ticket`: a paid event which requires a ticket
 
+#### type
+
+type is an open field, so any value is permitted, but this is a list of common/known types:
+
+- concert
+- theatre
+- meetup
+- museum
+- museum_railway
+- model_railway
+- guided_tour
+- workshop
+- ...
+
 #### Category enum values
 
 * `MUSIC`: concerts or other events where the main focus is music
 * `TECH`: event with technology as the focus
 * `ART`: art exhibitions, comedy, theater, ...
 
+#### RecurrenceType enum values
+
+* `REGULARLY`: events that happen about once a week or more often over the period of multiple months
+* `RARELY`: events that occur about once a month throughout the year
+* `ONCE`: events that occur once a year or more rarely (e.g. Christmas/Easter specials). When the same event is repeated
+  e.g. 3 times (for example so that more people can register), but only once a year, it should still be tagged as ONCE.
+
+Examples:
+
+* There is a special event with oldtimers on a museum railway that takes 2 days, but only happens once per year.
+    * should be tagged as ONCE
+* There is a special event with oldtimers on a museum railway that takes 2 days, but only happens once per year.
+    * should be tagged as ONCE
+* during Summer from June to September every Thursday there is a special train
+    * should be tagged as REGULARLY
+* a specific meetup happens about once per month or every two months
+    * should be tagged as RARELY
+
+#### tags values
+
+tags is an open field, so any value is permitted, but this is a list of common/known tags:
+
+- ...
+- TODO
+
+#### Image json
+
+```json
```

Comment:
we really have to talk about this change, because this changes our fundamental data model @Yolgie